index.js

```diff
@@ -188,7 +190,7 @@ Compiler.prototype.compileMessage = function(message, root) {
   return res;
 };
 
-module.exports = function(filename, model) {
+module.exports = function(filename, model,allow_additional_props) {
   var compiler = new Compiler(filename);
-  return compiler.compile(model);
+  return compiler.compile(model,allow_additional_props);
```

Comment:
Keep in mind that `model` is optional, so the third argument could become the second. I'd use something like

 ```javascript
if (typeof model === 'object') {
   opts = model;
   model = null;
}
```